### PR TITLE
New whitepaper - introduction

### PR DESCRIPTION
**Pull request by tpelliet**
- New intro added
- New whitepaper architecture added
- Old whitepaper removed

- Missing: Introduction images

### CODE REVIEW

File: docs/whitepaper/introduction.md [hunk 1]
```diff
@@ -4,202 +4,128 @@ sidebar_position: 2
...
+However, despite the omnipresence of data and algorithms, our knowledge remains limited. Why is this? **Because data and algorithms are not shared.**, essentially because of deficits of trust and incentives.
 
-OKP4 is built using the Cosmos SDK and is fundamentally a multichain infrastructure project. Our core value proposition relies on interchain applications leveraging existing networks inside and outside of the Cosmos ecosystem, such as Akash, Juno, Osmosis, Cheqd, Stargaze, Filecoin, Arweave, Ethereum and many more...
+This is where the decentralized web steps in, capable of reshuffling the cards in terms of knowledge creation and dissemination. The decentralized web holds the key for a fair and trusted infrastructure to share any existing dataset and algorithms, while tokens give innovating tools to incentivize collaboration. Within this context, OKP4 sets out to capitalize on the virtues of both Web 2.0 and Web 3.0 with the goal of propelling the knowledge economy to the next level.
```
Comment (MdechampG) [inline docs/whitepaper/introduction.md:29]:
```suggestion
This is where the decentralized web steps in, capable of reshuffling the cards in terms of knowledge creation and dissemination. The decentralized web holds the key for a fair and trusted infrastructure to share any existing datasets and algorithms, while tokens give innovating tools to incentivize collaboration. Within this context, OKP4 sets out to capitalize on the virtues of both Web 2.0 and Web 3.0 with the goal of propelling the knowledge economy to the next level.
```

File: docs/whitepaper/introduction.md [hunk 2]
```diff
@@ -4,202 +4,128 @@ sidebar_position: 2
...
-Each dataset and service interacts within to one or many Data Spaces according to the access given by the data and service providers.
-The outputs of the distributed workflows are leveraged by Applications to provide value to the end users.
+Ethereum expanded decentralization principles by introducing programmable smart contracts, enabling the development of numerous decentralized applications (dApps). However, Ethereum's broad scope comes with its own limitations. As a general-purpose platform, it may not always provide the most efficient solution for every use case.
 
-Anyone can create or customize Data Spaces and Applications using **OKP4 Development Kit**. It allows anyone to build and design with as much freedom as possible, as simply as possible. Developers, data scientists and communities at large can:
+Given the sensitivity of the data and knowledge sector, it's crucial that the infrastructure is as resilient and neutral as possible. Moreover, due to the intricacy of this field, there's a specific design requirement that fosters sharing while managing these complexities efficiently.
```
Comment (MdechampG) [inline docs/whitepaper/introduction.md:97]:
```suggestion
Given the sensitivity of the data and knowledge sector, the infrastructure must be as resilient and neutral as possible. Moreover, due to the intricacy of this field, there's a specific design requirement that fosters sharing while managing these complexities efficiently.
```

File: docs/whitepaper/introduction.md [hunk 3]
```diff
@@ -4,202 +4,128 @@ sidebar_position: 2
...
 
-## Network participants
+- Decentralized storage solutions like [Arweave](https://www.arweave.org/) or [Filecoin](https://filecoin.io/) offer secure and robust data storage, crucial for any data and knowledge management system.
+- [The Graph](https://thegraph.com/), a decentralized indexing solution, facilitates efficient data retrieval from blockchains through a GraphQL API, simplifying developer interaction with blockchain data.
+- [Kyve](https://www.kyve.network/), a decentralized data validation network, ensures data integrity and reliability, both essential for data management.
+- [Akash](https://akash.network/), a decentralized marketplace to buy and sell computing resources
+- [Secret Network](https://scrt.network/), offering decentralized privacy-preserving solutions such as Homomorphic Encryption
+- Other notable initiatives to valorize data include for exemple [Ocean Protocol](https://oceanprotocol.com/) specializing in data markets, [DIMO](https://dimo.zone/) incentivizing vehicule data sharing or [Hivemapper] incentivizing mapping data collection
```
Comment (MdechampG) [inline docs/whitepaper/introduction.md:106]:
```suggestion
- Other notable initiatives to valorize data include for exemple [Ocean Protocol](https://oceanprotocol.com/) specializing in data markets, [DIMO](https://dimo.zone/) incentivizing vehicule data sharing or [Hivemapper] incentivizing mapping data collection.
```

File: docs/whitepaper/introduction.md [hunk 4]
```diff
@@ -4,202 +4,128 @@ sidebar_position: 2
...
 
-- **Applications** also have custom governance. It can be as simple as a centralized front-end to consume workflow outputs, or more decentralized through community-run applications using only decentralized storage and computation resources.
-- **Datasets & services** are sovereign to the data and services providers. Providers (individuals or companies) can define general permissions/requirements for their digital resources to be used in many Data Spaces, but also can hand-pick the Data Spaces (and align themselves with the Data Space's rules) in which they want to contribute.
+From a technical standpoint, OKP4 offers a unique solution, deftly combining the virtues of openness and decentralization with the adaptive potential of interoperability and modularity.
```
Comment (MdechampG) [inline docs/whitepaper/introduction.md:129]:
```suggestion
From a technical standpoint, OKP4 offers a unique solution, deftly combining the virtues of **openness** and **decentralization** with the adaptive potential of **interoperability** and **modularity**.
```

**Review by ccamel — APPROVED** [review]

File: docs/whitepaper/introduction.md [hunk 5]
```diff
@@ -4,202 +4,136 @@ sidebar_position: 2
...
 
-OKP4 is an open interoperable smart contracts platform for highly customized rules, governance mechanisms and business models to align interests between participants and build useful distributed applications.
+However, despite the omnipresence of data and algorithms, our knowledge remains limited. Why is this? **Because data and algorithms are not shared**, essentially because of deficits of trust and incentives.
 
-Anyone can create custom ecosystems called [Data Spaces](https://gaia-x.eu/what-is-gaia-x/core-elements/data-spaces/). In each Data Space, people & companies can share data, algorithms, software, storage and computation according to Data Space's rules.
+This is where the decentralized web steps in, capable of reshuffling the cards in terms of knowledge creation and dissemination. The decentralized web holds the key for a fair and trusted infrastructure to share any existing dataset and algorithms, while tokens give innovating tools to incentivize collaboration. Within this context, OKP4 sets out to capitalize on the virtues of both Web 2.0 and Web 3.0 with the goal of propelling the knowledge economy to the next level.
```
Comment (amimart) [inline docs/whitepaper/introduction.md:29]:
```suggestion
This is where the decentralized web steps in, capable of reshuffling the cards in terms of knowledge creation and dissemination. The decentralized web holds the key for a fair and trusted infrastructure to share any existing dataset and algorithms, while tokens give innovating tools to incentivize collaboration. Within this context, OKP4 sets out to capitalize on the virtues of both Web 2 and Web 3 with the goal of propelling the knowledge economy to the next level.
```

I think Web3 and Web3.0 have different meanings

**Review by amimart — CHANGES_REQUESTED** [review]

**Review by MathieuConte — APPROVED** [review]

**Review by MdechampG — APPROVED** [review]

**Pull request merged**